doc/Development_Documentation/19_Development_Tools_and_Details/09_Cache/README.md

```diff
@@ -115,7 +115,7 @@ $uri = "http://www.pimcore.org/...";
 $cacheKey = md5($uri);
 if(!$data = \Pimcore\Cache::load($cacheKey)) {
  
-    $httpClient = \Pimcore\Tool::getHttpClient();
+    $httpClient = \Pimcore::getContainer()->get('pimcore.http_client');
```

Comment:
No. Suggest DI in the docs pls.

Reply:
One shall code as if `Pimcore::getContainer()` doesn‘t exist. 😏